ddp-workspace/projects/ddp-sdk/src/lib/services/logging.service.ts

```diff
@@ -12,6 +12,7 @@ type Logger = (message?: any, ...optionalParams: any[]) => void;
 @Injectable()
 export class LoggingService {
     private readonly LOG_SOURCE = 'Logging';
+    hello: Observable<void> = of();
```

Comment:
I guess, that was for debug purposes only.

Reply:
Ah! Garbage made it into the commit (was trying to figure out what type of() returns). Removed.

Reply:
1190fe8b1fb47ead674ad5e4c9bee8d8b544602e